### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by kbergin**
2017->2019

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 Human Cell Atlas Authors, https://humancellatlas.org
+Copyright (c) 2019 Human Cell Atlas Authors, https://humancellatlas.org
```
Comment (mckinsel) [inline LICENSE]: Think you want a range or list here: `2017, 2018, 2019`

Comment (kbergin) [reply LICENSE]: Huh that does seem possible. I just assumed it should be current. Perhaps @TimothyTickle knows better than I do.

Comment (kbergin) [reply LICENSE]: @TimothyTickle do you happen to know the answer here?

Comment (rexwangcc) [reply LICENSE]: I don't know the answer but did some googling: https://softwareengineering.stackexchange.com/a/210491

Comment (ambrosejcarr) [inline LICENSE]:
```suggestion
Copyright (c) 2017-2019 Human Cell Atlas Authors, https://humancellatlas.org
```
Based on Rex and Marcus's comments, the suggested change appears to be a correct approach.

**Comment by nikellepetrillo** [comment]
Closing this and will update in the new WARP repo: https://github.com/broadinstitute/warp

**Pull request closed without merging**